src/index.html

```diff
@@ -1,22 +1,447 @@
+            href="+1 234 5555-55-55"
+            class="menu__call-to-order"
+          >
+            Call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="main__content">
+        <div class="container">
+          <section
+            class="recommended"
+            id="recommended"
+          >
+            <h2 class="section-title section-title--align--left">
+              Recommended
+            </h2>
+            <div class="recommended__products">
+              <article class="product recommended__product">
+                <a
+                  href="#"
+                  class="product__link"
+                >
+                  <img
+                    src="images/products/beosound-a5.png"
+                    alt="BEOSOUND A5"
+                    class="product__photo"
+                  />
+                </a>
+                <h3 class="product__title">BEOSOUND A5</h3>
+                <p class="product__category">
+                  Huge sound anywhere – in the home or outdoors. Beosound A5 is
+                  a Wi-Fi and Bluetooth speaker with long-lasting battery and
+                  wireless charging built in.
+                </p>
+                <p class="product__price">$ 1299.00</p>
+              </article>
+
+              <article class="product recommended__product">
+                <a
+                  href="#"
+                  class="product__link"
+                >
+                  <img
+                    src="images/products/beoplay-hx.png"
+                    alt="BEOPLAY HX"
+                    class="product__photo"
+                  />
+                </a>
+                <h3 class="product__title">BEOPLAY HX</h3>
+                <p class="product__category">
+                  Wireless noise-cancelling headphones. Listen in comfort
+                  whether you're on a call, working from home or taking a moment
+                  for yourself.
+                </p>
+                <p class="product__price">$ 499.00</p>
+              </article>
+
+              <article class="product recommended__product">
+                <a
+                  href="#"
+                  class="product__link"
+                >
+                  <img
+                    src="images/products/beosound-theatre.png"
+                    alt="BEOSOUND THEATRE"
+                    class="product__photo"
+                  />
+                </a>
+                <h3 class="product__title">BEOSOUND THEATRE</h3>
+                <p class="product__category">
+                  Four patent-pending acoustic innovations in one sleek soundbar
+                  envelop you in sound from every angle. Made to fit any TV, and
+                  last for generations.
+                </p>
+                <p class="product__price">$ 3199.00</p>
+              </article>
+            </div>
+          </section>
+
+          <section
```

Comment:
Make sure to add the hover effect to increase the category images, as you did to the recommended section.